src/adapter/src/catalog/config.rs

```diff
@@ -132,6 +135,16 @@ impl Default for ClusterReplicaSizeMap {
                     workers: scale.into(),
                 },
             );
+
+            inner.insert(
```

Comment:
The new SIZE type , along with all the other types in this file, are defaults that are only used for testing. In the cloud, a completely separate set of SIZEs is installed and used.

Reply:
I'd feel more confident if we had this behind a `if testing`

Reply:
this entire thing is the default when executing bin/environmentd unless the entire SIZE list is overriden with a command-line option, which is what happens in the cloud
[13:24](https://materializeinc.slack.com/archives/D04LPD3UB8X/p1681986264993489)
there is no easy way to distinguish "test" from "non-test" invocations of bin/environmentd at this time